cfgov/jinja2/v1/index.html

```diff
@@ -180,6 +180,63 @@ <h3 class="h1 m-card_heading">97% of consumers</h3>
         <div class="wrapper">
             <div class="content_main content_main__home">
                 <h2>Latest updates</h2>
+
+                {% if flag_enabled('AUTOMATIC_LATEST_UPDATES') %}
+
+                <div class="content-l">
+                    {% import 'macros/time.html' as time %}
+                    {% import 'macros/category-slug.html' as category_slug %}
+                    {% for post in latest_updates %}
+
+                    {% if loop.index0 % 2 == 0 %}
```

Comment:
Consider making this `loop.index % 2 == 1` so you don't have to think about the index starting from 0 here, but starting from 1 below.

Reply:
Done in b900f64